cfme/tests/automate/test_common_methods.py

```diff
@@ -104,22 +76,42 @@ def test_vm_retire_extend(request, testing_vm, soft_assert, retire_extend_button
     testing_vm.set_retirement_date(retirement_date)
     wait_for(lambda: testing_vm.retirement_date != 'Never', message="retirement date set")
     set_date = testing_vm.retirement_date
+    vm_retire_date_fmt = VM.RETIRE_DATE_FMT.pick(appliance.version)
+
     if not BZ(1419150, forced_streams='5.6').blocks:
-        soft_assert(set_date == retirement_date.strftime(pick(VM.RETIRE_DATE_FMT)),
+        soft_assert(set_date == retirement_date.strftime(vm_retire_date_fmt),
                     "The retirement date '{}' did not match expected date '{}'"
-                    .format(set_date, retirement_date.strftime(pick(VM.RETIRE_DATE_FMT))))
+                    .format(set_date, retirement_date.strftime(vm_retire_date_fmt)))
 
     # Create the vm_retire_extend button and click on it
-    retire_extend_button()
+    grp_name = "grp_{}".format(fauxfactory.gen_alphanumeric())
+    grp = ButtonGroup(
+        text=grp_name,
+        hover=grp_name,
+        type=ButtonGroup.VM_INSTANCE
+    )
+    request.addfinalizer(lambda: grp.delete_if_exists())
+    grp.create()
+    btn_name = "btn_{}".format(fauxfactory.gen_alphanumeric())
+    button = Button(
+        group=grp,
+        text=btn_name,
+        hover=btn_name,
+        system="Request",
+        request="vm_retire_extend"
+    )
+    request.addfinalizer(lambda: button.delete_if_exists())
+    button.create()
+
+    toolbar.select(grp.text, button.text)
```

Comment:
Create a view with the dropdown here in the test function:
```python
# ...
# ...

class TestDropdownView(View):
    group = Dropdown(grp.text)

view = appliance.create_view(TestDropdownView)  # or how is it
view.group.select_item(button.text)
```

That should do the trick.

Reply:
@mfalesni actually that was not a Dropdown but BootstrapTreeview. I used click_path to get what I needed.